legendary_eureka.py

```diff
@@ -31,9 +33,120 @@ def get_jobs(self):
 
 
 class Glassdoor(Resource):
+
+    JOB_TYPES = ['fulltime', 'parttime', 'contract', 'internship', 'temporary', 'apprenticeship', 'entrylevel']
+
     def __init__(self):
         Resource.__init__(self, 'glassdoor')
 
-    # TODO
-    def get_jobs(self):
-        pass
+    # TODO: wrap requests.post parts with try / except.
```

Comment:
@Avielyo10 i would like to know how to do it right, and also how to unittest the except case with connection exception.

Reply:
See [pytest](https://docs.pytest.org/en/latest/)